lib/cron.dart

```diff
@@ -120,8 +132,8 @@ class _Cron implements Cron {
     if (_closed) return;
     if (_timer != null || _schedules.isEmpty) return;
     final now = DateTime.now();
-    final ms = _millisecondsPerMinute -
-        (now.millisecondsSinceEpoch % _millisecondsPerMinute);
+    final ms = _millisecondsPerSecond -
```

Comment:
I think one more change would be nice to have here: prior to this, scheduling the timer to tick every minute was a low-overhead thing, however every second seems to be unnecessary if you happen to have schedules to trigger only every hour or so...

Could you please change this to have both? Tick every second only if there are schedules with seconds, otherwise tick every minute.

Reply:
> I think one more change would be nice to have here: prior to this, scheduling the timer to tick every minute was a low-overhead thing, however every second seems to be unnecessary if you happen to have schedules to trigger only every hour or so...
> 
> Could you please change this to have both? Tick every second only if there are schedules with seconds, otherwise tick every minute.

I changed it